GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -187,7 +187,15 @@ public final class ColorsXMLFileParser: ColorsFileParser {
                       namespaceURI: String?, qualifiedName qName: String?) {
       guard elementName == ColorsXMLFileParser.colorTagName else { return }
       guard let colorName = currentColorName, let colorValue = currentColorValue else { return }
-      parsedColors[colorName] = try? parse(hex: colorValue) ?? UInt32(0)
+
+      do {
+        parsedColors[colorName] = try parse(hex: colorValue)
+      } catch let e as ColorsParserError {
+        fatalError(e.description)
+      } catch {
+        fatalError("error: unknown color parser error")
```

Comment:
I see 3 possible solutions here:

* We use `(try? parse(hex: colorValue)) ?? UInt32(0)` and don't error. Seems like a bad solution as there won't be any error message to the user
* We do as you suggest, with `parser.abortParsing()` and store the error in some property to check it at the end outside of the parser delegate.
* We transform the `var parsedColors = [String: UInt32]()` in the delegate into a `var parsedColors = [String: ColorResult]()` with `enum ColorResult { case color(UInt32), error(ColorsParserError) }` so we can store either a valid color UInt32 or an error then handle that at the end of the `parseFile` function.

The first solution is easier a d quicker to implement. The second solution would allow us to continue parsing and report _all_ errors at the end. And still even allow the output file to be generated even (with partial output) if we want to (but we better match the behavior for all color parsers then)